pkg/scheduler/framework/plugins/noderesources/least_allocated_test.go

```diff
@@ -407,11 +407,16 @@ func TestLeastAllocatedScoringStrategy(t *testing.T) {
 				return
 			}
 
+			status := p.(framework.PreScorePlugin).PreScore(ctx, state, test.requestedPod, test.nodes)
+			if !status.IsSuccess() {
+				t.Errorf("PreScore is expected to return success, but didn't. Got status: %v", status)
+			}
+
 			var gotScores framework.NodeScoreList
 			for _, n := range test.nodes {
 				score, status := p.(framework.ScorePlugin).Score(ctx, state, test.requestedPod, n.Name)
 				if status.Code() != test.wantStatusCode {
-					t.Errorf("unexpected status code, want: %v, got: %v", test.wantStatusCode, status)
+					t.Errorf("Score is expected to return success, but didn't, want: %v, got: %v", test.wantStatusCode, status)
```

Comment:
For this one, the previous message is suitable. 

```suggestion
					t.Errorf("unexpected status code, want: %v, got: %v", test.wantStatusCode, status.Code())
```